static-tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+GOFMT_FILES=$(gofmt -l .)
+if [ -n "${GOFMT_FILES}" ]; then
+  printf >&2 'gofmt failed for the following files:\n%s\n\nplease run "gofmt -w ." on your changes before committing.\n' "${GOFMT_FILES}"
+  # exit 1
+fi
+
+GOLINT_ERRORS=$(golint ./... | grep -v "Id should be")
+if [ -n "${GOLINT_ERRORS}" ]; then
+  printf >&2 'golint failed for the following reasons:\n%s\n\nplease run 'golint ./...' on your changes before committing.\n' "${GOLINT_ERRORS}"
+  # exit 1
+fi
+
+GOVET_ERRORS=$(go tool vet *.go 2>&1)
```

Comment:
When I run this with the `-v` flag it only prints out `rita.go`.  Was the intention to only check this file or should all go files under rita be checked?  e.g. `go tool vet $(find . -name '*.go' | grep -v '/vendor/')`

Reply:
Go vet is being run against all files now.